lib/angular_wait_barrier.ts

```diff
@@ -0,0 +1,106 @@
+import {SimpleWebDriverClient} from './simple_webdriver_client';
+import {WebDriverCommand} from './webdriver_commands';
+import {WebDriverLogger} from './webdriver_logger';
+import {WebDriverBarrier} from './webdriver_proxy';
+
+let angularWaits = require('./client_scripts/wait.js');
+
+export class AngularWaitBarrer implements WebDriverBarrier {
```

Comment:
Should have a JSDoc comment block about the `AngularWaitBarrer` class. Also, is it supposed to be `AngularWaitBarrer` or `AngularWaitBarrier` (spellling?)